webapp/tests/karma/ts/modules/tasks/tasks.component.spec.ts

```diff
@@ -278,4 +303,161 @@ describe('TasksComponent', () => {
       expect(component.listTrackBy(0, false)).to.equal(undefined);
     });
   });
+
+  describe('lineage and breadcrumbs', () => {
+    const bettysContactDoc = {
+      _id: 'user',
+      parent: {
+        _id: 'parent',
+        name: 'CHW Bettys Area',
+      },
+    };
+    const taskDocs = [
+      { _id: '1', emission: { _id: 'e1', dueDate: '2020-10-20' }, forId: 'a', owner: 'a' },
+      { _id: '2', emission: { _id: 'e2', dueDate: '2020-10-20' }, forId: 'b', owner: 'b' },
+    ];
+    const taskLineages = [
+      {
+        _id: 'a', lineage: [
+          {name: 'Amy Johnsons Household'},
+          {name: 'St Elmos Concession'},
+          {name: 'Chattanooga Village'},
+          {name: 'CHW Bettys Area'},
+        ]
+      },
+      {
+        _id: 'b', lineage: [
+          {name: 'Amy Johnsons Household'},
+          {name: 'St Elmos Concession'},
+          {name: 'Chattanooga Village'},
+        ]
+      }
+    ];
+
+    it('it should not change the reports lineages if the connected user is online only', async() => {
```

Comment:
Targets! no reports :)

And space after `async` as it's reserved word. 
```suggestion
    it('should not alter target lineages if user is online only', async () => {
```